### ISSUE TITLE
Multiple configuration files example

### ISSUE THREAD

**Issue by Kipriz** [grails3book/feedback#7]
The book has a great start. One thing that I miss in Grails 3 is external configuration files in  ``grails.config.locations``
I think a lot of Grails 2.x developers are looking for a way to split configuration to a few files and load environment specific property files.
It would be nice to have good examples in the book how to use multiple configuration files in Grails 3. Gradle, Spring Boot and Grails itself have their own ways to provide configuration and properties files. It is no so easy to understand how they play together.

**Comment by erichelgeson** [comment]
Agreed! Thanks for the feedback. I'll include this in the next release.

**Comment by erichelgeson** [comment]
Also, if you haven't seen this, give it a try to get similar to grails 2 behavior: http://plugins.grails.org/plugin/grails/external-config
